app/lib/nhs/immunisations_api.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module NHS::ImmunisationsAPI
+  class PatientNotFound < StandardError
+  end
+
+  class << self
+    def record_immunisation(vaccination_record)
```

Comment:
Is it worth putting a guard here to make sure we're only sending vaccination records that were recorded directly in Mavis?

Reply:
And should we have a similar guard to only send records for vaccinations that were given? Or should these checks go somewhere else?

Reply:
Oh and also somewhere we need to limit this to just specific vaccination types.

Reply:
Discussed with Misha, and these restrictions will be put in place at the point the job is enqueued.